README.md

```diff
@@ -19,27 +19,41 @@ At the end of the day, it's about [_writing less code_](https://youtu.be/GnO7D5U
 
 Head over to [our company Handbook](https://mui-org.notion.site/Why-MUI-d8b8c142a6a44e3aa963f26edf4e03db) to learn more!
 
-## Challenge summary
+## Introduction
 
 Given that the Design Engineer position sits right at the intersection between engineering and design, this challenge was tailored to elevate your skills in both areas.
 Our challenge is to **develop a stacked Snackbar** component.
 
 Here's a reference design:
 
-![Snackbars example design](public/snackbar-reference.gif)
+<img alt="Snackbars example design" src="./public/snackbar-reference.gif" width="465" height="392" />
```

Comment:
That's so much better ⎯ thanks! How did you record this git, just out of curiosity?

Reply:
Thank you!